nixos/modules/system/boot/systemd/initrd.nix

```diff
@@ -430,7 +428,7 @@ in {
         "${cfg.package}/lib/systemd/systemd-fsck"
         "${cfg.package}/lib/systemd/systemd-hibernate-resume"
         "${cfg.package}/lib/systemd/systemd-journald"
-        (lib.mkIf needMakefs "${cfg.package}/lib/systemd/systemd-makefs")
+        "${cfg.package}/lib/systemd/systemd-makefs"
```

Comment:
Honestly lets also add `systemd-growfs`. Somebody is gonna show up with a usecase where they want to grow a filesystem in initrd.  (E.g. appliance images that never switch-root; but need an extra data volume).

Reply:
It should be quite easy to add if/once this usecase shows up, not gonna do it in this PR ;-)